llvm/test/Bitcode/attributes.ll

```diff
@@ -613,6 +618,7 @@ define void @initializes(ptr initializes((-4, 0), (4, 8)) %a) {
 ; CHECK: attributes #52 = { nosanitize_bounds }
 ; CHECK: attributes #53 = { sanitize_realtime }
 ; CHECK: attributes #54 = { nosanitize_realtime }
```

Comment:
Note that when we strip out this attribute, all of your function "numbers" will be affected, I will try to get this merged asap

Reply:
Thanks - updated as part of rebasing